--- ../setup.py	2024-09-05 05:54:52.262670867 +0000
+++ setup.py	2024-09-05 05:55:03.652760274 +0000
@@ -83,7 +83,7 @@
     author_email='tom@tomchristie.com',  # SEE NOTE BELOW (*)
     packages=find_packages(exclude=['tests*']),
     include_package_data=True,
-    install_requires=["django>=3.0", 'backports.zoneinfo;python_version<"3.9"'],
+    install_requires=["django>=3.0"],
     python_requires=">=3.6",
     zip_safe=False,
     classifiers=[
