--- ../PKG-INFO	2024-09-02 11:34:21.319324401 +0000
+++ PKG-INFO	2024-09-02 11:34:30.507206066 +0000
@@ -26,7 +26,7 @@
 Requires-Dist: incremental>=22.10.0
 Requires-Dist: twisted-iocpsupport<2,>=1.0.2; platform_system == 'Windows'
 Requires-Dist: typing-extensions>=4.2.0
-Requires-Dist: zope-interface>=5
+Requires-Dist: zope.interface>=5
 Provides-Extra: all-non-platform
 Requires-Dist: twisted[conch,http2,serial,test,tls]; extra == 'all-non-platform'
 Provides-Extra: all_non_platform
